### ISSUE TITLE
Help needed. Devices don’t show up. 

### ISSUE THREAD

**Issue by SCOTTS948** [tuya/tuya-homebridge#338]
My devices stopped working. I discovered my trial period on the Tuya site expired. I requested an extension which was granted. I rebooted Homebridge, but my devices still won’t show up in HomeKit. I’ve read of others having success after renewing the trial. I need help getting my devices back in the Home app.

**Comment by pks0** [comment]
Same here. I even refreshed everything - new Tuya IoT cloud project, app linking, homebridge reset… Devices still don’t show up. The Tuya API response for getting devices is blank but no error. Seems some update on Tuya cloud has made plug-in incompatible.

For the time being, I have pulled in some devices to SmartThings via Tuya app linking and then I am able to manage them in Home app via homebridge-Smartthings plugin. Devices that show support for ST can be managed this way.

**Comment by sciencegirl100** [comment]
Same here, looks like I get an error 2406 in the console and it says my config is incorrect. I tipple checked my setting and they all match the Tuya platform details.

**Comment by InDoubt** [comment]
Trying to reveal a Strip Light by Feit Electric to not avail.
Strip Light is recognized and operable by the app Tuya Smart and automatically visible in the app eve (iOS 16.5.1 - iPhone XsMax)
Followed all instructions to add and configure the plug in in Homebridge running in a MacMini server (macOS Monterrey latest)

Config:

    "options": {
        "username": "m12345m@gmail.com",
        "password": "M12345",
        "accessId": "sry1234567890n4",
        "accessKey": "4e12345678901234567890bb",
        "lang": "en",
        "projectType": "2",
        "appSchema": "tuyaSmart",
        "countryCode": 1,
        "debug": false
    },
    "name": "TuyaPlatformFeitShopLight",
    "platform": "TuyaPlatformFeitShopLight",
    "_bridge": {
        "username": "12:34:56:78:90:11",
        "port": 11111
    }
}

Homebridge interface shows:

TuyaPlatformFeitShopLight (set as child bridge)
Bridge Paired (positively paired)

In the TUYA IoT Platform> Cloud the Tuya App has been added and linked:
Name: m12345m@gmail.com | Typeapp账号
Device Management Method:Automatic Link
Device Name: Feit Electric Strip Device ID: eb12345678901234567890mj Product: Strip Light Device Status: Online   Device Type: Real Device Activation Time: 2023-07-07 11:12:06 | Device Permission: Read  Operation: Unlink (this last is an option greyed out)

When trying to add the device to Homekit, I use the camera to read the code generated by Homebridge/TuyaPlatformFeitShopLight child brige.
The process goes smooth up to the end and still the Strip Light doesn't show up anywhere in Home app.

I noticed that the app "eve" (in the iPhone too) can see a device : "TuyaPlatformFeitShopLight CEEB (Bridge)" but scroling through the details shows:
"Bridged Accessories: NONE"

Am I doing this all wrong?
Missing something?
Not possible at all?

I appreciate some help!
Thanks in advance!

J
